src/component/form-fields.html

```diff
@@ -1,10 +1,17 @@
 <template>
 
-  <form-field
+  <form-field repeat.for="element of schema"
+    if.bind="hasKeyProp(element)"
     value.bind="model[element.key]"
     message.bind="messages[element.key]"
-    element.bind="element"
-    repeat.for="element of schema">
+    element.bind="element">
+  </form-field>
+
+  <form-field repeat.for="element of schema"
+    if.bind="!hasKeyProp(element)"
+    value.bind="model"
+    message.bind="messages[element.key]"
+    element.bind="element">
```

Comment:
Instead of copying the same element can't you just do:
```
<form-field repeat.for="element of schema"
     if.bind="!hasKeyProp(element)"
     value.bind="hasKeyProp(element) ? model[element.key] : model)"
     message.bind="messages[element.key]"
     element.bind="element">
```